modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/Comments/details.js

```diff
@@ -123,6 +163,27 @@ $(function (){
                 data: "creationTime",
                 orderable: true,
                 dataFormat: "datetime"
+            },
+            {
+                width: "10%",
+                title: l("Status"),
+                orderable: false,
+                data: "isApproved",
+                render: function (data, type, row) {
+                    var icons = ''
+
+                    if (data === null) {
+                        icons = '<i class="fa-solid fa-hourglass-start"></i>';
+                    } else if (typeof data === "boolean") {
+                        if (data) {
+                            icons = '<i class="fa-solid fa-check" style="color: #63E6BE;"></i>';
```

Comment:
Instead of hard-coded color codes, use CSS files to apply styles. Ask to @armgnunlu if you need help.